bundles/org.openhab.binding.netatmo/src/main/java/org/openhab/binding/netatmo/internal/handler/capability/WeatherCapability.java

```diff
@@ -39,7 +39,7 @@ public WeatherCapability(CommonInterface handler) {
     @Override
     protected List<NAObject> updateReadings(WeatherApi api) {
         try {
-            return List.of(api.getStationData(handler.getId()));
+            return List.of(api.getStationData(handler.getId(), readonly));
```

Comment:
Just for my understanding: If read-only it means that user doesn't have any favorite stations in addition to the user's own stations?

Reply:
readonly will be set to true in the returned data for a favorite or guest station.
Here you have to pass true as second parameter when you want the data associated to a station you do not own, either a favorite oir guest station.